manim/__init__.py

```diff
@@ -2,8 +2,7 @@
 from . import config
 from . import constants
 
-
-def main():
+def main():    
```

Comment:
A per PEP8, please leave two blank lines here

Reply:
but not two blank spaces!